assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -75,8 +82,11 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
     );
   }, []);
 
+  const titleFormatted = useCallback((rawTitle) => {
+    return rawTitle === __('Auto Draft', 'web-stories') ? '' : rawTitle;
+  }, []);
+
```

Comment:
Is there any benefit of `useCallback` here?
This can be moved outside the component for best-case performance-wise (_if we care about it so much_).

Reply:
Good suggestion. But I go with disable button when the title is not valid.